src/mainboard/raptor-cs/talos-2/romstage.c

```diff
@@ -3,12 +3,40 @@
 #include <console/console.h>
 #include <cpu/power/istep_8_9.h>
 #include <cpu/power/istep_8_10.h>
+#include <cpu/power/istep_13.h>
 #include <program_loading.h>
+#include <device/smbus_host.h>
+#include <lib.h>	// hexdump
+#include <spd_bin.h>
 
 void main(void)
```

Comment:
At this point this function should be called mainboard_get_spd since we do nothing mainboard-specific besides SPD.

The main should be moved to src/soc/ibm/power9/romstage.c or something like that with all the step calls and SPD decoding.